LICENSE.txt

```diff
@@ -201,674 +201,6 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 
---------------------------------------------------------------------------------
```

Comment:
I reviewed all these items prior to deleting them to confirm they referred to code that only is in the arrow repository (not arrow-datafusion)